exercises/concept/arcade-high-score/src/test/java/ArcadeHighScoreTest.java

```diff
@@ -0,0 +1,90 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ArcadeHighScoreTest {
+
+    private final ArcadeHighScore arcadeHighScore = new ArcadeHighScore();
```

Comment:
You probably want to create a new instance for every test using `@BeforeEach` to make sure the SUT doesn't retain any state from a previous test, this makes the tests easier to reason about, also for students!